src/main/webapp/app/hooks/useTextareaAutoHeight.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect } from 'react';
+import { InputType } from 'zlib';
```

Comment:
The InputType is wrong, not suppose to come from zlib, but from reactstrap. 

I let the useTextareaAutoHeight hook determine if it should resize depending on input type, instead of putting that logic in the calling component. If its not text area, then the hook will not do anything.